inputs/kiln-input.js

```diff
@@ -259,7 +259,7 @@ export default class KilnInput {
   * @return {string|number|object|array}
   */
   value(val) {
-    if (val) {
+    if (val != null) {
```

Comment:
Please use `!==` instead
```suggestion
    if (val !== null) {
```

Reply:
With this condition we can avoid `null` and `undefined` at the same time